src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task{
```

Comment:
Missed a space between 'Task' and '{'. Otherwise, overall layout is pretty good!